cmd/secret_test.go

```diff
@@ -0,0 +1,55 @@
+package cmd
+
+import (
+	"os"
+
+	"github.com/jarcoal/httpmock"
+)
+
+func Example_secret_add() {
+	pwd, _ := os.Getwd()
+
+	HeritageConfigFilePath = pwd + "/test/test-barcerola.yml"
+
+	app := newTestApp(SecretCommand)
+	endpoint := os.Getenv("BARCELONA_ENDPOINT")
+
+	testArgs := []string{
+		"bcn", "secret", "add", "-n", "bcn-test",
+		"-v", "test", "-d", "staging"}
+
+	httpmock.Activate()
+	defer httpmock.DeactivateAndReset()
+
+	httpmock.RegisterResponder("POST", endpoint+"/v1/districts/staging/ssm_parameters",
+		httpmock.NewStringResponder(200, ""))
+
+	app.Run(testArgs)
+
+	// Output:
+	// success to set bcn-test
+}
+
+func Example_secret_delete() {
+	pwd, _ := os.Getwd()
+
+	HeritageConfigFilePath = pwd + "/test/test-barcerola.yml"
+
+	app := newTestApp(SecretCommand)
+	endpoint := os.Getenv("BARCELONA_ENDPOINT")
+
+	testArgs := []string{
+		"bcn", "secret", "delete", "-n", "bcn-test", "-d", "staging"}
+
+	httpmock.Activate()
+	defer httpmock.DeactivateAndReset()
```

Comment:
Can we please also have a test with names that have slashes in them? This is a common thing we use, so we should ensure future tests do not break them.